js/components/close-caption-multi-audio-menu/closeCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,145 @@
+      skinConfig: this.props.skinConfig
+    }
+  },
+
+  /**
+   * Getting a list of available audio tracks
+   * @return {object} - list of available audio tracks and selected track
+   * @private
+   */
+  _getMultiAudio: function () {
+    var multiAudio = {
+      list: [],
+      selected: null
+    };
+
+    if(!!this.props.controller.state.multiAudio) {
+      if(Array.isArray(this.props.controller.state.multiAudio.tracks)) {
+        multiAudio.list = this.props.controller.state.multiAudio.tracks;
+        multiAudio.selected = _.find(this.props.controller.state.multiAudio.tracks, function (track) {
+          return track.enabled === true;
+        });
+      }
+    }
+
+    return multiAudio;
+  },
+
+  /**
+   * Getting the list of available languages for subtitles
+   * @return {object} list of available languages for subtitles and selected language
+   * @private
+   */
+  _getClosedCaptions: function () {
+    var closeCaptions = {
+      list: [],
+      selected: null
+    };
+
+    if (this.props.controller.state.closedCaptionOptions) {
+      if (this.props.controller.state.closedCaptionOptions.availableLanguages &&
+        this.props.controller.state.closedCaptionOptions.availableLanguages.languages &&
+        Array.isArray(this.props.controller.state.closedCaptionOptions.availableLanguages.languages)) {
+        closeCaptions.list = this.props.controller.state.closedCaptionOptions.availableLanguages.languages;
+      }
+
+      if (this.props.controller.state.closedCaptionOptions.language) {
+        closeCaptions.selected = this.props.controller.state.closedCaptionOptions.language;
+      }
+    }
+
+    return closeCaptions;
+  },
+
+  handleSelectCC: function (id) {
+    this.closeCaptions.selected = id;
+    this.setState({
+      closeCaptions: this.closeCaptions
+    });
+
+    if (this.props.controller && this.props.controller.onClosedCaptionChange) {
+      this.props.controller.onClosedCaptionChange('language', id);
+    }
+  },
+
+  handleSelectMA: function (id) {
+    var selectedElement = null;
+
+    this.multiAudio.list.forEach(function (el) {
+      if (el.id === id) {
+        el.enabled = true;
+        selectedElement = el;
+      } else {
+        el.enabled = false;
+      }
+    });
+    this.multiAudio.selected = selectedElement;
+
+    this.setState({
+      multiAudio: this.multiAudio
+    });
+
+    if (this.props.controller && this.props.controller.setCurrentAudio) {
+      this.props.controller.setCurrentAudio(id);
+    }
+  },
+
+  render: function () {
+    var menuClass = "oo-cc-ma-menu";
+    var columnLeftClass = "oo-column-left";
+    var columnRightClass = "oo-column-right";
+    var separatorClass = "oo-separator";
+
+    if(this.state.multiAudio.list.length === 0){
+      columnLeftClass = columnLeftClass + " hidden";
+      separatorClass = separatorClass + " hidden";
+      columnRightClass = columnRightClass + " full";
+    }
+
+    if(this.state.closeCaptions.list.length === 0) {
```

Comment:
Would it be simpler to just set a single class on the container which affects the child element classes instead of setting a special class for each element? Such as:
```
.oo-cc-ma-menu.oo-empty .oo-column-left {
  // left column style when the menu is empty
}
.oo-cc-ma-menu.oo-empty .oo-column-right {
  // right column style when the menu is empty
}
```